src/Button.tsx

```diff
@@ -186,43 +187,39 @@ function Button(props) {
     color && { backgroundColor: color }, // custom backgroundColor
     flex && { flex }, // flex width
     height && { height }, // custom height
-    borderWidth && { borderWidth },
-    borderColor && { borderColor },
     marginSpacing,
     paddingSpacing,
     style
   ]);
 
-  const backgroundColor = StyleSheet.flatten(buttonStyles).backgroundColor;
-
   if (disabled) {
+    const backgroundColor = StyleSheet.flatten(buttonStyles).backgroundColor;
```

Comment:
This can be extracted outside the if condition as it is used multiple times. See the last PR merge